modules/silicon_design/scripts/build/images/provision/environment.yml

```diff
@@ -13,14 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 channels:
+  - defaults
```

Comment:
curious why you needed this?

Reply:
is that different from `main`?

Reply:
im actually not sure it was just in the cfu playground env so i ported it over

Reply:
@proppy will double check